confluent_kafka/deserializing_consumer.py

```diff
@@ -43,59 +43,57 @@ class DeserializingConsumer(_ConsumerImpl):
-    |                    |                 | Callable(KafkaError)                                |
-    |                    |                 |                                                     |
-    | error_cb           | callable        | Callback for generic/global error events. These     |
-    |                    |                 | errors are typically to be considered informational |
-    |                    |                 | since the client will automatically try to recover. |
-    +--------------------+-----------------+-----------------------------------------------------+
-    | log_cb             | logging.Handler | Logging handler to forward logs                     |
-    +--------------------+-----------------+-----------------------------------------------------+
-    |                    |                 | Callable(str)                                       |
-    |                    |                 |                                                     |
-    |                    |                 | Callback for statistics. This callback is           |
-    | stats_cb           | callable        | added to the application queue every                |
-    |                    |                 | ``statistics.interval.ms`` (configured separately). |
-    |                    |                 | The function argument is a JSON formatted str       |
-    |                    |                 | containing statistics data.                         |
-    +--------------------+-----------------+-----------------------------------------------------+
-    |                    |                 | Callable(ThrottleEvent)                             |
-    | throttle_cb        | callable        |                                                     |
-    |                    |                 | Callback for throttled request reporting.           |
-    +--------------------+-----------------+-----------------------------------------------------+
-
-    .. _See Client CONFIGURATION.md for a complete list of configuration properties:
-        https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+    Notable DeserializingConsumer configuration properties(* indicates required field)
+
+    +----------------------+-----------------+-----------------------------------------------------+
+    | Property Name        | Type            | Description                                         |
+    +======================+=================+=====================================================+
+    | `bootstrap.servers*` | str             | Comma-separated list of brokers.                    |
+    +----------------------+-----------------+-----------------------------------------------------+
+    |                      |                 | Client group id string.                             |
+    | `group.id*`          | str             | All clients sharing the same group.id belong to the |
```

Comment:
nit: the splat should probably be outside the ticks as not to confuse it as part of the literal.
But no need to fix now.